src/components/organisms/SectionContents/features.jsx

```diff
@@ -5,6 +5,12 @@ import * as React from 'react';
 //** Atoms */
 import ModalBtn from "../../atoms/ModalButton"
 
+//** Helpers */
+//** Personalization */
+import Text from "../../helper/Text";
+import { renderToString } from 'react-dom/server';
+import ReactHtmlParser from 'react-html-parser'; 
+
 /**
  * Section: Features
  */
```

Comment:
This is disgusting drug-induced crap, and should die
To put it with Linus words.